shapeshift/lib
# feat: idle-investor final

Created the Fetching system and Interaction layer for Idle Finance opportunities

To test:

1. Link locally to web
2. Enable IdleFinance in the Feature Flags
3. Check if Idle Opportunities shows in Defi/Earn section
4. Deposit/Withdraw in both Best-Yield and Junior/Senior tranches
5. Claim rewards from deposited Best-Yield strategies
6. Deposited Idle opportunities should appear in Defi/Overview section

NOTE:

Make sure to execute `yarn generate` to build the `generatedAssetData.json` inside `packages/asset-service` and link the `@shapeshiftoss/asset-service` to web.
